packages/sync/src/conflicts/ConflictResolutionStrategy.ts

```diff
@@ -10,3 +10,8 @@ import { ConflictResolutionData } from "./ConflictResolutionData";
  */
 export type ConflictResolutionStrategy =
   (operationName: string, server: ConflictResolutionData, client: ConflictResolutionData) => ConflictResolutionData;
+
+export interface IResolver {
```

Comment:
Missing docs. 
All end user objects needs to be documented.
Wrong folder (it should be in api)